.github/workflows/lint.yml

```diff
@@ -45,18 +48,25 @@ jobs:
             experimental/node_modules
             */*/node_modules
             experimental/*/*/node_modules
-          key: lint-${{ matrix.package-group }}-${{ runner.os }}-${{ matrix.node_version }}-${{ hashFiles('**/package.json') }}-20211009.2
+          key: lint-${{ matrix.package-group }}-${{ runner.os }}-${{ matrix.node_version }}-${{ hashFiles('**/package.json') }}-20211216.2
 
       # On a cache miss, install dependencies
       - name: Bootstrap
         if: steps.cache.outputs.cache-hit != 'true'
         working-directory: ${{ matrix.package-group }}
         run: |
           npm install --ignore-scripts
-          npx lerna bootstrap --no-ci --ignore-scripts --hoist --nohoist='zone.js' -- --only=dev
```

Comment:
Is it required now to run scripts and install all dependencies? These options were there to speed up the run before.